diff --git BUILD.bazel BUILD.bazel
new file mode 100644
index 0000000..5778b45
--- /dev/null
+++ BUILD.bazel
@@ -0,0 +1 @@
+cc_library(name = "enet_lib", srcs = glob(["*.c"]), hdrs = glob(["include/enet/*.h"]), includes = ["include"], visibility = ["//visibility:public"])
diff --git MODULE.bazel MODULE.bazel
new file mode 100644
index 0000000..b23e1e8
--- /dev/null
+++ MODULE.bazel
@@ -0,0 +1 @@
+module(name = "enet", version = "1.0")
